Jenkinsfile

```diff
@@ -21,15 +15,20 @@ node {
                 GIT_REPOSITORY = scmVars.GIT_URL
                 GIT_BRANCH_NAME = scmVars.GIT_BRANCH
                 GIT_COMMIT_ID = scmVars.GIT_COMMIT
+
+                TIMESTAMP = new java.text.SimpleDateFormat("yyMMddHHmmss").format(new Date())
             }
 
             stage("Create version number") {
-                ARTIFACT_VERSION = (GIT_BRANCH_NAME + "-" + GIT_COMMIT_ID).replaceAll("[^\\w-]", "_")
-                sh "git branch ${ARTIFACT_VERSION}"
+                if (env.BRANCH_NAME == 'master') {
+                    ARTIFACT_VERSION = "release-${TIMESTAMP}"
+                } else {
+                    ARTIFACT_VERSION = "pr-${GIT_BRANCH_NAME.replaceAll('/', '_')}-${TIMESTAMP}"
+                }
             }
 
             stage("Build opsgenie-heartbeat-proxy image") {
-                DOCKER_IMAGE_NAME = "opsgenie-heartbeat-proxy:" + ARTIFACT_VERSION;
+                DOCKER_IMAGE_NAME = "949398260329.dkr.ecr.eu-west-1.amazonaws.com/opsgenie-heartbeat-proxy:${ARTIFACT_VERSION}";
```

Comment:
Are you sure with `${ARTIFACT_VERSION}` ?

Reply:
What do you mean? If the question is about if the syntax is valid: yes, this is how string interpolation is done in Groovy.